packages/core/types/zapier.custom.d.ts

```diff
@@ -43,6 +43,7 @@ export interface Bundle<InputData = { [x: string]: any }> {
     isTestingAuth: boolean;
     limit: number;
     page: number;
+    timezone: string | undefined;
```

Comment:
```suggestion
    timezone: string | null;
```

Since the backend returns `None`.

Reply:
Thanks for catching that! Indeed `None` converts to `null` via JSON.